client/components/domains/search-filters/style.scss

```diff
@@ -178,17 +178,21 @@
 body.is-section-signup.is-white-signup {
 	$accent-blue: #117ac9;
 
+	.search-filters__popover,
 	.search-filters-extensions__popover {
-		@include break-medium {
-			transform: translateX( 65px );
-		}
```

Comment:
Thanks so much for addressing this Omar! It's much better already but I think that we still have to include the transform in `breakpoints > mobile` (see screenshot) since the modal isn't aligned with the search bar anymore. WDYT? 

<img width="887" alt="Screenshot 2021-06-18 at 12 15 48" src="https://user-images.githubusercontent.com/12104589/122546244-0cb28b00-d02f-11eb-9f0b-164f1013df20.png">

Reply:
The modal wasn't aligned with the field before this change, and I'm not sure if we should touch that because it affects [other places as well](https://github.com/Automattic/wp-calypso/pull/53779#issuecomment-863904718).

Reply:
Let's leave it as is :)